util/logutil/log.go

```diff
@@ -110,6 +129,14 @@ func InitLogger(cfg *LogConfig) error {
 		return errors.Trace(err)
 	}
 
+	// initialize general log logger
+	GeneralLogLogger, err = newGeneralLogLogger()
```

Comment:
I think we need to check whether the general log config is opened or not first?🙈

Reply:
Yes, but there seems not a global `enable_general_log` config for now.

Reply:
Agree with @jyz0309 . It is highly risky to just throw a new logger (as well as new goroutine) in a global functions. It may break existing settings or assumption.

Reply:
@tisonkun You made it more clear. Thanks and I will make a new function like `InitGeneralQueryLogger()` instead of insert it in `InitLogger()` which is referenced more than ten times.

Reply:
@dragonly if it makes sense for your requirement. You can let the function return a cancel function as `context.Background` so that also fixes the goroutine leaks.